generator/traffic_generator.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "traffic/traffic_info.hpp"
+
+#include "std/string.hpp"
+#include "std/vector.hpp"
+
+namespace traffic
+{
+bool GenerateTrafficKeysFromDataFile(string const & mwmPath, bool forceRebuild);
```

Comment:
On the one hand, this seems to be a common way to write section generators.
On the other hand, forceRebuild seems to never be used.

Removed.